test/logic/stake_logic_test.rb

```diff
@@ -167,6 +168,44 @@ def setup
     assert_equal 7.5, stake_pool.reload.average_validators_commission
   end
 
+  test "count average_score" do
```

Comment:
split this test to smaller ones and describe what is being tested or add some comments here